src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java

```diff
@@ -305,6 +361,26 @@ public void validate(FailureCollector collector, Schema inputSchema) {
           }
         }
       }
+
+      if (dlpOverrideEnabled) {
+        if (dlpHost == null || dlpHost.isEmpty()) {
```

Comment:
Any more validation for hostname? check for valid ip or web address? Not necessary if its too much work

Reply:
The rules should validate characters for hostnames, IPv4 and IPv6, so I'll leave it out for now. I'll add trim() to the dlpHost though.